File: Desarrollo_Movil/AndroidStudio/Ejemplo2Canvas/app/src/main/java/com/example/ejemplo2canvas/MainActivity.java
package com.example.ejemplo2canvas;

import android.os.Bundle;
import android.app.Activity;

public class MainActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Lienzo l = new Lienzo(this);
        setContentView(l);
    }
}

File: Desarrollo_Movil/AndroidStudio/KotlinPDF3/app/src/main/java/com/example/kotlinpdf3/MainActivity.kt
package com.example.kotlinpdf3

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText
import android.widget.Toast

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val btnScan = findViewById<Button>(R.id.inputBtn)
        val input = findViewById<EditText>(R.id.input)

        btnScan.setOnClickListener {
            val a = input.text!!.length
            Toast.makeText(this,"La cadena mide: " + a, Toast.LENGTH_LONG).show()
        }
    }
}
